src/components/book-form.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const BooksForm = () => (
```

Comment:
This should be a class based component so we can have as state and bind the form input to the values in state.